BlotoutAnalytics/Utilities/BOServerDataConverter.m

```diff
@@ -57,14 +58,11 @@ +(NSDictionary*)recordAppInformation {
       int timeoOffset = [BOAUtilities getCurrentTimezoneOffsetInMin];
       [appInfo setObject:[NSNumber numberWithInt:timeoOffset] forKey:@"timeZoneOffset"];
       
-      //UIWebView Depriciated , need to handle it with WKWebView
       dispatch_async(dispatch_get_main_queue(), ^{
-      UIWebView* webView = [[UIWebView alloc] initWithFrame:CGRectZero];
-     
-      NSString* userAgent = [webView stringByEvaluatingJavaScriptFromString:@"navigator.userAgent"];
-          
+      NSString *userAgent = [[WKWebView new] valueForKey:@"userAgent"];
```

Comment:
is `WKWebView` always available or is it possible that this would be not set (if sdk is called in some other thread for example)?

Reply:
this is explicitly called on the main thread, so that would not be an issue.